### PR TITLE
Np 8361 search project with organization

### PR DESCRIPTION
**Pull request by svenngjeto**

### CODE REVIEW

File: docs/cristin-proxy-swagger.yaml [hunk 1]
```diff
@@ -46,6 +46,14 @@ paths:
...
             pattern: '.+'
           example: 'Helse'
+        - name: organization
+          in: query
+          description: Search projects by organization id.
+          required: false
+          schema:
+            type: string
+            pattern: '.+'
+          example: 'https://api.dev.nva.aws.unit.no/cristin/organization/20202.0.0.0'
```
Comment (torarnet) [inline docs/cristin-proxy-swagger.yaml:56]: Are you using an uri as a query parameter? Is it not better to only use the last part 20202.0.0.0? I dont think it is wise to use 2 uris in the same request

File: cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java [hunk 2]
```diff
@@ -128,4 +132,32 @@ public static URI createNvaPositionId(String code) {
         return attempt(() -> new URI(positionBase.getScheme(), positionBase.getHost(), positionBase.getPath(), code))
                 .orElseThrow();
     }
+
+    /**
+     * Convenience method to encode a uri.
```
Comment (torarnet) [inline cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java:137]: to encode or decode?

**Review by torarnet — COMMENTED** [review]
See comments.

File: docs/cristin-proxy-swagger.yaml (hunk 1 above)
Comment (svenngjeto) [reply docs/cristin-proxy-swagger.yaml:56]: It's the URI which is the full and unambiguous id which precisely and persistent identifies the Organization - and we don't have to operate on partial identifers. 
Maybe how and what to use for id should be discussed for the whole project?

File: cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java (hunk 2 above)
Comment (svenngjeto) [reply cristin-commons/src/main/java/no/unit/nva/utils/UriUtils.java]: To decode or not to decode - that's the question...

**Review by torarnet — APPROVED** [review]

**Review by brinxmat — APPROVED** [review]

**Pull request merged**